code/modules/reagents/chemistry_machinery/autodispenser.dm

```diff
@@ -17,6 +17,7 @@
 	icon = 'icons/obj/structures/machinery/science_machines.dmi'
 	icon_state = "autodispenser"
 	active_power_usage = 40
+	health = 300
```

Comment:
add a define schema for structure HP, e.g. code/_DEFINES/structure.dm  #DEFINE STRUCTURE_HEALTH_WEAK 50 , #DEFINE STRUCTURE_HEALTH_BASE 100 , #DEFINE STRUCTURE_HEALTH_REINFORCED 300